### PR TITLE
CC-56 [FE] Replace PNG pictures into SVG in whole app

### PR DESCRIPTION
**Pull request by ArtsemiLasyi**

### CODE REVIEW

File: src/utils/IconMatcher.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                return <CollaborationIcon />;
+            case SideBarIds.Schedule:
+                return <ScheduleIcon />;
+            case SideBarIds.Contacts:
+                return <ContactsIcon />;
+            case SideBarIds.Donate:
+                return <DonateIcon />;
+        }
+    }
+}
```
Comment (17Sergey) [inline src/utils/IconMatcher.tsx]: Empty line after this please

File: src/components/common/Icons/ChurchIcon.tsx
Comment (17Sergey) [inline src/components/common/Icons/ChurchIcon.tsx:1]: I have a general question, why did you decide to use this approach of creating JSX-svg components?
You might take a look at our ```Heading.tsx``` component. It uses a library 'react-svgmt'
```
                        <SvgLoader path={HeadingShapeLeft}>
                            <SvgProxy selector="rect" fill={shapeColor} />
                        </SvgLoader>
```
It allows us to pass a path for svg and manipulate with its children by selectors. Using this approach we could leave all the Sidebar icons in for example ```assets/icons``` directory and create icon components that would just work with those ```SvgLoader, SvgProxy```. What do you think about that?

**Comment by 17Sergey** [comment]
These svg icons don't match our style. Maybe we need to look for more matching icons.

File: src/utils/IconMatcher.tsx (hunk 1 above)
Comment (PavelPyasecky) [inline src/utils/IconMatcher.tsx]: Do not forget about empty ending line.

**Comment by PavelPyasecky** [comment]
I've just looked through our icons. They are pretty good but the line width is so large. Do we have opportunity to change it? If no, could we have apply consistent line width of all icons?

**Comment by PavelPyasecky** [comment]
In addition I've found strange behavior during opening sidebar: the left icons padding depends on sidebar state.

**Comment by PavelPyasecky** [comment]
@ArtsemiLasyi @17Sergey I would like to propose you such library of icons as https://ant.design/components/icon.
We can use next components instead of current:
```
<BankOutlined />
<ScheduleOutlined />
<TeamOutlined />
<EnvironmentOutlined />
<DollarOutlined />
```
They are well decorated and consistent. Let's use it.

**Review by PavelPyasecky — APPROVED** [review]
Icons look wonderful! May be one of the best free solutions:)

**Comment by ArtsemiLasyi** [comment]
> In addition I've found strange behavior during opening sidebar: the left icons padding depends on sidebar state.

@PavelPyasecky, fixed :)

**Comment by ArtsemiLasyi** [comment]
> @ArtsemiLasyi @17Sergey I would like to propose you such library of icons as https://ant.design/components/icon. We can use next components instead of current:
> 
> ```
> <BankOutlined />
> <ScheduleOutlined />
> <TeamOutlined />
> <EnvironmentOutlined />
> <DollarOutlined />
> ```
> 
> They are well decorated and consistent. Let's use it.

After the discussion we decided to use more suitable free svg icons

**Review by 17Sergey — APPROVED** [review]

**Pull request merged**